### DISCUSSION TITLE
Power management options

### DISCUSSION THREAD

**Discussion by FabioFrmg** [Lolliedieb/lolMiner-releases#1722] [General]
Hello guys.
I'm new here and i'm migrating from ETH mining. The new coin i plan to mine suggests using LOL miner.
But I have a doubt...

Does Lol miner have video card power management options?

Do these options work with AMD cards?

One of the main needs of those using AMD cards is that the mining software manages the clock of the memories and VGA core clock, if possible, the VGA mining power factor also.
This is because AMD drivers do not keep saved settings and in case of power failures. 
Not everyone is adept at performing BIOS MOD on their boards, for compatibility reasons.
If these resources are not available, would it be possible to implement them?
Thank you very much in advance.

**Comment by esippert** [comment]
https://github.com/Lolliedieb/lolMiner-releases/blob/master/README.md#lolminer-158

**Reply by FabioFrmg** [reply]
Only Core and mem clocks? 
Works in AMD cards? 
What about fan control? Power control? etc?
